5-infrastructure/business_unit_1/development/development.auto.tfvars

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+parent_folder              = "953533373736"
+boa_gke_project_id         = "prj-bu1-d-boa-gke-ecb0"
+boa_ops_project_id         = "prj-bu1-d-boa-ops-040f"
+boa_sec_project_id         = "prj-bu1-d-boa-sec-7cd1"
+boa_sql_project_id         = "prj-bu1-d-boa-sql-4f49"
+gcp_shared_vpc_project_id  = "prj-d-shared-base-f12c"
+terraform_service_account  = "terraform-boa-sa@prj-d-shared-base-f12c.iam.gserviceaccount.com"
```

Comment:
Lets explicitly set all these as examples following example-foundation conventions so it is clear to the user that this needs to me changed.
https://github.com/terraform-google-modules/terraform-example-foundation/blob/b1704781ba4a004de5ba8917feded5361259ce12/3-networks/common.auto.example.tfvars#L19